src/goose/brokers/redis/commands.clj

```diff
@@ -155,6 +155,22 @@
 (defn del-from-list [conn list element]
   (wcar* conn (car/lrem list 1 element)))
 
+(defn list-position-multiple [conn list elements]
+  (wcar* conn (doseq [e elements]
+                (car/lpos list e))))
+
+(defn del-from-list-multiple [conn list elements]
+  (wcar* conn (doseq [e elements]
+                (car/lrem list 1 e))))
+
+(defn del-from-list-and-enqueue-front-multiple [conn list elements]
```

Comment:
NOTE: Since the number of arguments are same for `del-from-list-and-enqueue-front` given a single element vs multiple elements same function with mutiple arity can't be used.
Hence different functions are defined with name ending with multiple to imply this.
This is also repeated for `list-position` and `del-from-list`.